shell/server/notifications-server.js

```diff
@@ -0,0 +1,90 @@
+// Sandstorm - Personal Cloud Sandbox
+// Copyright (c) 2016 Sandstorm Development Group, Inc. and contributors
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+logActivity = function (grainId, identityId, event) {
```

Comment:
This seems to be introducing a new global function which you depend on the existence of in other files.  Let's try not to do that any more.

An alternative would be to placed this function in `shell/imports/server/notifications.js`, export it from that file (`export { logActivity };`), and in the files where you use the function, at the top level, add a line `import { logActivity } from "/imports/server/notifications.js";`